src/System.Runtime.InteropServices.RuntimeInformation/src/System/Runtime/InteropServices/RuntimeInformation/OSPlatform.cs

```diff
@@ -8,12 +9,43 @@ namespace System.Runtime.InteropServices
     {
         private readonly string _osPlatform;
 
+        public static OSPlatform Android { get; } = new OSPlatform("ANDROID");
```

Comment:
For such members to actually be exposed, they'd also need to be added to the .cs file in the ref folder, as that's what influences the reference assembly:
https://github.com/migueldeicaza/corefx/blob/23055b101d38f16a84dfe23a3b0a05807381926c/src/System.Runtime.InteropServices.RuntimeInformation/ref/System.Runtime.InteropServices.RuntimeInformation.cs